Assets/Scripts/Game/Items/ItemCollection.cs

```diff
@@ -348,16 +348,18 @@ public DaggerfallUnityItem GetItem(int index)
         /// </summary>
         /// <param name="itemGroup">Item group.</param>
         /// <param name="itemIndex">Template index.</param>
-        /// <param name="priorityToConjured">Prefer (short lived) conjured items.</param>
+        /// <param name="allowEnchantedItem">Include enchanted items.</param>
+        /// <param name="allowQuestItem">Include quest items.</param>
+        /// <param name="preferConjured">Prefer (short lived) conjured items.</param>
         /// <returns>An item of this type, or null if none found.</returns>
-        public DaggerfallUnityItem GetItem(ItemGroups itemGroup, int itemIndex, bool priorityToConjured = false)
+        public DaggerfallUnityItem GetItem(ItemGroups itemGroup, int itemIndex, bool allowEnchantedItem = true, bool allowQuestItem = true, bool preferConjured = false)
```

Comment:
May i suggest to use an overload instead, to preserve compatibility with mods?

Reply:
Would that do?
C# didn't let me create a GetItem(ItemGroups, int, bool) next to my new method because of the ambiguity it would cause...

Reply:
Ah, I see, interesting implementation, and minimal runtime overhead. I'll probably need some more time to totally wrap my head around the consequences.
In any case, thank you for the investigation and provided solutions!